### DISCUSSION TITLE
Google Cloud installer updates status

### DISCUSSION THREAD

**Discussion by seattlebrighton** [NightscoutFoundation/xDrip#3156] [General]
If Google cloud installer will be updated to install Nightscout 15, can a row be added to this: https://navid200.github.io/xDrip/docs/Nightscout/GC_ReleaseNotes.html
So users can track progress, or see if dev help is needed?
I'd this today in progress already?
Thank you for your incredible work!

**Comment by seattlebrighton** [comment]
Sorry, meant to ask if this update is in progress.

**Comment by Navid200** [comment]
Work is in progress.
There will be a release to the dev branch today or tomorrow.

**Comment by Navid200** [comment]
There is a new release in the dev branch, which updates to version 15.0.2 of Nightscout.
In a few days, if everything looks good, we will update the stable branch as well.

This explains everything about the branches: https://navid200.github.io/xDrip/docs/Nightscout/Branches.html

**Comment by QuixNL** [comment]
Am i correct that the stable branch is still on 14.2.6? and that i need to switch to the dev branch if i want to use nightscout 15.0.2?

**Reply by Navid200** [reply]
I'm not aware of anything in the stable release that stops it from uploading to Nightscout 15.0.2.

**Reply by QuixNL** [reply]
That's strange if i look at nightscout status i am on /jamorham/nightscout-vps/vps-1. and just did an update platform but nightscout version is still n 14.2.6. did is miss something?

**Reply by Navid200** [reply]
Can you post a screenshot of your Google Cloud Nightscout status page?

**Reply by QuixNL** [reply]
Yes sure 
<img width="392" alt="Scherm­afbeelding 2024-01-21 om 15 30 14" src="https://github.com/NightscoutFoundation/xDrip/assets/59391906/c54850a0-bb9b-463e-9532-876f6b370794">

i hope it is showing...

**Reply by Navid200** [reply]
Why do you think it is still running 14.2.6?

**Reply by QuixNL** [reply]
that is what is see on my nightscout site:
<img width="1560" alt="Scherm­afbeelding 2024-01-21 om 15 34 31" src="https://github.com/NightscoutFoundation/xDrip/assets/59391906/f020b684-7f4b-485e-a51e-a9a38f5f498c">

**Reply by Navid200** [reply]
OK

Would you please restart the server?
https://navid200.github.io/xDrip/docs/Nightscout/Restart.html

**Reply by QuixNL** [reply]
After reboot i had to fix the 502 bad gateway by running fase 1 again and another reboot.
I probably missed these steps in the step by step guide.

It is working and on version 15.0.2

Thanks for the support

very happy with the work you have done!

**Reply by Navid200** [reply]
Great,

I confess that a lot has been done to this setup.  And it was obvious to me that there was no way for me to remember it all.
So, starting a long time ago, my intent has been to create a keymap that shows what is causing a flag and how to address it.
https://navid200.github.io/xDrip/docs/Nightscout/StatusKey.html

But, some changes do not create a flag until a restart!  This is obviously a problem.
So, I am going to add several pointers throughout the documentation encouraging restarts.
Restarting the server is harmless and can sort out a lot.  But, currently, I am finding I have not emphasized that enough throughout the guide.

Thanks for letting me see it again.
I will fix it.
